src/components/canditadeMiniCard/candidateMiniCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import './candidateMiniCard.scss';
+import {Card, CardContent, Divider, Grid, Paper, Typography} from "@material-ui/core";
+type CandidateMiniCardProps={
+    timeZon:string,
+    op?:string
+}
+const  CandidateMiniCard: React.FC<CandidateMiniCardProps>= ({timeZon,op}) => {
+    const date = new Date("2012-05-24");
```

Comment:
Why custom Date === '2012-05-24' ?

Reply:
It is just an example for me how works intl.